src/accessibilityStatement/AccessibilityStatement.tsx

```diff
@@ -31,9 +32,10 @@ function AccessibilityStatement(): React.ReactElement {
   return (
     <div className={styles['page-wrapper']}>
       <Header />
-      <div className={styles['container']}>
+      <PageMeta title={t('footer.accessibility')} />
```

Comment:
This should probably have its own title in the translations, and not reuse a translation for another use.

Reply:
Added a new translation